internal/rbd/replicationcontrollerserver.go

```diff
@@ -495,6 +611,10 @@ func (rs *ReplicationServer) DemoteVolume(ctx context.Context,
 
 	// demote image to secondary
 	if mirroringInfo.Primary {
+		err = disableMirroringOnDummyImage(rbdVol)
```

Comment:
We need to ensure dummy image is primary (even if split-brain) for the enable/disable schedule to not error out. IOW, the dummy image can be in a split-brain scenario forever, but needs to be primary for these operations.

Reply:
Ignore @BenamarMk pointed out that we create different dummy images per cluster, so that would always be primary for the local cluster allowing enable/disable calls.